lib/wunderbar/html-methods.rb

```diff
@@ -189,8 +189,8 @@ def method_missing(name, *args, &block)
       name = name.to_s.gsub('_', '-')
 
       if flag != '!'
-        if String === args.first
-          if not block and args.first =~ /[>&]/
+        if String === args.first and args.first.respond_to? :html_safe?
```

Comment:
I suppose this was the error that was introduced.  The correct way to have removed html_safe support would have been to assume that html_safe is false and remove the entire if check.

Reply:
AFAICT that would mean dropping the entire paragraph.
I'll do another PR